renovate.json

```diff
@@ -1,7 +1,24 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["github>konflux-ci/mintmaker//config/renovate/renovate.json"],
+  "extends": ["github>konflux-ci/mintmaker/config/renovate/renovate.json"],
   "tekton": {
     "schedule": ["at any time"]
-  }
+  },
+  "baseBranches": ["master", "/^foreman-.*$/"],
+  "packageRules": [
+    {
+      "matchBaseBranches": ["/^foreman-.*$/"],
+      "description": "Disable all updates on foreman branches",
+      "enabled": false
+    },
+    {
+      "matchBaseBranches": ["/^foreman-.*$/"],
+      "description": "Only allow updates to certain files",
+      "includePaths": [
+        "rpms.lock.yaml",
+        "build/Dockerfile",
+        ".tekton/**"
```

Comment:
These are specific and sometimes very specific to a project (see the path to a Dockerfile above). How about using [`enabledManagers`](https://docs.renovatebot.com/configuration-options/#enabledmanagers)?

Reply:
Exactly these three as an example:
* tekton
* dockerfile
* rpm-lockfile
https://github.com/konflux-ci/mintmaker/blob/ed27b4872df93a19641348240f243065adcd90d9/config/renovate/renovate.json#L27-L29